lib/web_ui/dev/driver_manager.dart

```diff
@@ -0,0 +1,145 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:io' as io;
+
+import 'package:meta/meta.dart';
+import 'package:path/path.dart' as pathlib;
+import 'package:web_driver_installer/chrome_driver_installer.dart';
+import 'package:web_driver_installer/safari_driver_runner.dart';
+
+import 'chrome_installer.dart';
+import 'common.dart';
+import 'environment.dart';
+import 'utils.dart';
+
+/// [DriverManager] implementation for Chrome.
+///
+/// This manager can be used for both MacOS and Linux.
+class ChromeDriverManager extends DriverManager {
+  ChromeDriverManager(String browser) : super(browser);
+
+  Future<void> _installDriver() async {
+    if (_browserDriverDir.existsSync()) {
+      _browserDriverDir.deleteSync(recursive: true);
+    }
+
+    _browserDriverDir.createSync(recursive: true);
+    temporaryDirectories.add(_drivers);
+
+    io.Directory temp = io.Directory.current;
+    io.Directory.current = _browserDriverDir;
+
+    // TODO(nurhan): https://github.com/flutter/flutter/issues/53179
+    final String chromeDriverVersion = await queryChromeDriverVersion();
+    ChromeDriverInstaller chromeDriverInstaller =
+        ChromeDriverInstaller.withVersion(chromeDriverVersion);
+    await chromeDriverInstaller.install(alwaysInstall: true);
+    io.Directory.current = temp;
+  }
+
+  /// Driver should already exist on LUCI as a CIPD package.
+  ///
+  /// Throw an error if directory does not exists.
```

Comment:
dartdocs should start with the sentence that explains what the function does, so I'd swap the two paragraphs.

https://dart.dev/guides/language/effective-dart/documentation#prefer-starting-function-or-method-comments-with-third-person-verbs

Reply:
done